metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -297,4 +297,15 @@ entities:
       - queryProperties
       - querySubjects
       - status
+  - name: schema
+    category: core
+    keyAspect: schemaKey
+    aspects:
```

Comment:
The typical `schemaProperties` aspect may be missed here.
So we can set display name, external url, custom properties, ....